Cohort1/Medlista/androidApp/src/main/java/com/example/medlista/android/HomeScreen.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.medlista.android
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+
+class HomeScreen : ComponentActivity() {
```

Comment:
Wondering if this should be called "HomeScreenActivity" since I've always seen activities labeled that way but you may have researched the naming better than me. Just something I noticed. And maybe naming is different in compose.